crowdsec-docs/docs/appsec/rules.md

```diff
@@ -0,0 +1,136 @@
+---
+id: rules
+title: Rules
+sidebar_position: 3
+---
+
+<!--
+
+ describe the overall rule (waap rules) organization, relevant section.
+ explain the DSL
+ speak a bit about how to write yours and how to write waap-tests in the hub (kka)
+ how to debug them
+
+-->
+
+Rules are the core of the application security component. They are used to detect and block attacks.
```

Comment:
maybe point out how they differ from the parser+scenario, or the fact that the "rules" specifically applies to the AppSec context and are meant to analyse the request elements specifically